src/layout/AppLayout.tsx

```diff
@@ -10,13 +21,16 @@ interface Props {
 export function AppLayout({ pathName, children }: PropsWithChildren<Props>) {
   return (
     <>
-      <HeadMeta pathName={pathName} />
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>{`Liquid Staking | ${getHeadTitle(pathName)}`}</title>
+      </Head>
       <div
         className={`flex flex-col justify-between h-full min-h-screen w-full min-w-screen bg-gradient-radial`}
       >
         <Header pathName={pathName} />
         <main className="w-full">{children}</main>
-        <Footer />
+        <footer className="w-screen py-4 px-7">Empty footer</footer>
```

Comment:
We need to add a proper footer anyway. Let's keep it as a separate component.

Reply:
Done